model/src/wmgridmd.F90

```diff
@@ -2966,8 +2969,10 @@ SUBROUTINE WMGHGH
                               NLOC_OM   = NLOC_OM + 1
 ! Notes: check here that we are sufficiently dimensioned.
                               IF ( NLOC_OM .GT. NLMAX ) THEN
-                                 IF ( IMPROC.EQ.NMPERR ) WRITE (MDSE,1020)
-                                 CALL EXTCDE(1020)
+                                 IF ( IMPROC.EQ.NMPERR ) WRITE (MDSE,*)
```

Comment:
Revert to original formatted write statement and error call?

Reply:
Reverted!